vscode-extension/src/extension.ts

```diff
@@ -26,6 +32,46 @@ export function activate(context: vscode.ExtensionContext) {
   });
   context.subscriptions.push(setupCommand);
 
+  let shareCommand = vscode.commands.registerCommand(COMMANDS.SHARE, async () => {
+    // TODO: use vscode.window.activeTextEditor instead of hardcoding
+    const fileName: string = "/Users/rossdancraig/Projects/aiconfig/cookbooks/Gemini/gemini_demo.aiconfig.json";
```

Comment:
Should do this before landing

Reply:
cc @saqadri can you land final version where you had it updated?